clients/cloud/python/README.md

```diff
@@ -5,7 +5,7 @@ Produce messages to and consume messages from a Kafka cluster using [Confluent P
 
 # Prerequisites
 
-* [Confluent's Python Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-python) installed on your machine. Check that you are using version 1.0.0 or higher (e.g., `pip show confluent-kafka`).
+* [Confluent's Python Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-python) installed on your machine. Check that you are using version 1.4.2 or higher (e.g., `pip show confluent-kafka`).
```

Comment:
Python2 is no longer supported, so propose

```suggestion
* [Confluent's Python Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-python) installed on your machine. Check that you are using version 1.4.2 or higher (e.g., `pip3 show confluent-kafka`).
```

Reply:
This is similar to the `python` she-bang issue above.  I'm not familiar enough with the python ecosystem.  If we change the env above, I think this makes sense stating that we only support python3, but if we don't change above, then this seems confusing.

Reply:
Virtualenv should be used rather than relying on the system installation. This will simplify things as you can explicitly instruct it to run a particular interpreter. 

https://virtualenv.pypa.io/en/latest/user_guide.html#introduction

Reply:
🤦 , of course... Thanks @rnpridgeon .  I'm going to add some documentation to help users if this is new to them or have memory failure like I do.

Reply:
I've added some instructions for installing and running Virtualenv to the README